Automattic/wp-calypso
# Reader Lists: list followed tags

This PR adds a scrollable list of a Reader list's existing followed tags.

Fixes https://github.com/Automattic/wp-calypso/issues/637.

![screen shot 2015-11-25 at 16 01 54](https://cloud.githubusercontent.com/assets/17325/11402103/096c1396-938e-11e5-82b8-352d1a02e6f6.png)

### To test

Choose an existing Reader list that you own, and click 'Manage' in the sidebar.

Navigate to the 'Tags' tab, and ensure that any tags you've previously set up appear in the list.

<img width="1207" alt="screen shot 2015-12-14 at 09 15 40" src="https://cloud.githubusercontent.com/assets/17325/11769816/6414939e-a244-11e5-948e-11c1f81273c1.png">
